homeassistant/components/hassio.py

```diff
@@ -164,15 +216,17 @@ def send_command(self, command, method="post", payload=None, timeout=10):
             with async_timeout.timeout(timeout, loop=self.loop):
                 request = yield from self.websession.request(
                     method, "http://{}{}".format(self._ip, command),
-                    json=payload)
+                    json=payload, headers={
+                        X_HASSIO: os.environ.get('API_TOKEN')
```

Comment:
This environment variable should have a better name. API TOKEN can be for anything. Maybe `HASSIO_TOKEN` ?

Reply:
Will there always be an API TOKEN ? Or can it be that we're sending a header with a value of `None` ? If there can always be one, use `os.environ['API_TOKEN']` so it blows up when it is not there.

Reply:
Yes we can send this with None, ping give a response but other service calls will not work after the security layer is active.

I rename this: https://github.com/home-assistant/hassio/pull/303 but for Add-ons it will be `API_TOKEN`